inscriptions/ISic000335.xml

```diff
@@ -185,14 +190,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#MH">
+                <ab>
+                <lb n="1"/>Bictoria
+                <lb n="2"/>vixit an  
+                <lb n="3" break="no"/>nis <num value="4">IIII</num>
+                <lb n="4"/>meses <num value="9">VIIII</num>
```

Comment:
I would suggest that one would encode the erroneous/non-standard form of 'menses' thus:
`<choice><orig>meses</orig><reg>menses</reg></choice>`

Reply:
Further, you could do the same thing to Bictoria/Victoria in line 1?